frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavTransactionOptions.java

```diff
@@ -0,0 +1,78 @@
+package com.ncapdevi.fragnav;
+
+import android.support.annotation.AnimRes;
+import android.support.annotation.StyleRes;
+import android.support.v4.util.Pair;
+import android.view.View;
+
+import java.util.List;
+
+/**
+ *
+ */
+
+
+public class FragNavTransactionOptions {
+    List<Pair<View, String>> sharedElements;
+    @FragNavController.Transit
+    Integer transition;
+    @AnimRes
+    Integer enterAnimationId;
+    @AnimRes
+    Integer exitAnimationId;
+    @StyleRes
+    Integer transitionStyle;
+
+    private FragNavTransactionOptions(Builder builder) {
+        sharedElements = builder.sharedElements;
+        transition = builder.transition;
+        enterAnimationId = builder.enterAnimationId;
+        exitAnimationId = builder.exitAnimationId;
+        transitionStyle = builder.transitionStyle;
+    }
+
+    public static Builder newBuilder() {
+        return new Builder();
+    }
+
+    public static final class Builder {
```

Comment:
Might as well add all possible options now since someone is inevitably going to ask for them later:
```
addToBackStack
breadCrumbTitle
breadCrumbShortTitle
popEnterAnimation
popExitAnimation
```